ssl/build.info

```diff
@@ -32,11 +28,23 @@ SOURCE[../libssl]=\
         ssl_asn1.c ssl_txt.c ssl_init.c ssl_conf.c  ssl_mcnf.c \
         bio_ssl.c ssl_err.c ssl_err_legacy.c tls_srp.c t1_trce.c ssl_utst.c \
         record/ssl3_buffer.c record/ssl3_record.c record/dtls1_bitmap.c \
-        statem/statem.c record/ssl3_record_tls13.c record/tls_pad.c \
+        statem/statem.c record/ssl3_record_tls13.c \
         tls_depr.c $KTLSSRC
 IF[{- !$disabled{'deprecated-3.0'} -}]
-  SOURCE[../libssl]=s3_cbc.c  ssl_rsa_legacy.c
+  SOURCE[../libssl]=ssl_rsa_legacy.c
 ENDIF
+# For shared builds we need to include the libcrypto packet.c and sources
+# needed in providers (s3_cbc.c and record/tls_pad.c) in libssl as well.
+# We can either continue to do it like this, or export all the used
+# symbols so that libssl can use them like any other. Probably would do
```

Comment:
We don't have the option of _private_ exports.  I'd just remove that sentence.

Reply:
fixup pushed